content/api/commands/session.md

```diff
@@ -24,7 +24,8 @@ Enabling this flag does the following:
   and [`Cypress.session`](/api/cypress-api/session) API.
 - It adds the following new behaviors (that will be the default in a future
   major update of Cypress) at the beginning of each test:
-  - The page is cleared (by setting it to `about:blank`).
+  - The page is cleared (by setting it to `about:blank`). Disable this by
+    setting `testIsolation=legacy`.
```

Comment:
Can this link to the configuration doc or a doc that explains `testIsolation` and how/where to set it?